### PR TITLE
Add an internal reducer to store the last time a channel fetched posts

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Instead of using the last update_at from the posts that we have in the store, this creates an internal store that has the last time a channel fetched posts and that timestamp will be used to `getPostSince` that way we can avoid to miss messages.

### CODE REVIEW

File: src/actions/posts.js [hunk 1]
```diff
@@ -360,7 +360,18 @@ export function getPosts(channelId, page = 0, perPage = Posts.POST_CHUNK_SIZE) {
             return null;
         }
 
-        dispatch(batchActions([
+        const actions = [];
+        if (Object.keys(posts.posts).length === Posts.POST_CHUNK_SIZE) {
```
Comment (jwilander) [inline src/actions/posts.js]: Won't this set the timestamp if you scroll up and load more posts in the channel? I think we only want to set it if you don't have any posts in the channel yet unless I'm mistaken

**Comment by enahum** [comment]
If you scroll up it will use getPostBefore, right?

**Comment by jwilander** [comment]
On the webapp we only use that for permalink view, and use the paging here to get more posts

**Comment by enahum** [comment]
You are right, I'll fix it by checking that the page number is 0, but I'll do that over the weekend as I'm not at my desk

**Comment by jwilander** [comment]
Ok cool, that should work

**Comment by enahum** [comment]
@jwilander done

File: src/actions/posts.js [hunk 2]
```diff
@@ -360,7 +360,18 @@ export function getPosts(channelId, page = 0, perPage = Posts.POST_CHUNK_SIZE) {
             return null;
         }
 
-        dispatch(batchActions([
+        const actions = [];
+        if (page === 0 && Object.keys(posts.posts).length === Posts.POST_CHUNK_SIZE) {
```
Comment (jwilander) [inline src/actions/posts.js]: I don't think we need the length check do we? What happens in a newer channel with only a few posts?

Comment (enahum) [inline src/actions/posts.js]: Well at leadt in RN when that happens we call getPosts again

Comment (jwilander) [inline src/actions/posts.js]: Ok, up to you if you want to change it or not

**Pull request merged**